core/client.py

```diff
@@ -77,6 +77,12 @@ def train(self, client_data, model, conf):
 
         # TODO: One may hope to run fixed number of epochs, instead of iterations
         while completed_steps < conf.local_steps:
+            
+            if len(client_data) == 0:
```

Comment:
wondering why this can happen? I think we have already removed these zero-sample clients [here](https://github.com/SymbioticLab/FedScale/blob/master/core/client_manager.py#L47).

Reply:
that make sense, maybe I forget to set the args